app/frontend/src/components/minesattestations/Step3.vue

```diff
@@ -198,6 +198,45 @@
       </div>
     </div>
 
+    <div class="question-series">
+      <h3 class="question-head">9. Transportation of Workers</h3>
+      <div class="questions">
+        <p>Modes of transportation (choose all that apply)</p>
+        <div class="questions-check-group">
+          <v-checkbox
+            v-model="transportationSingleOccupant"
+            :readonly="reviewMode"
+            label="One person per vehicle"
+          ></v-checkbox>
```

Comment:
Could make the elements singletons instead of element pairs
`<v-checkbox ... />`